maskrcnn_benchmark/layers/batch_norm.py

```diff
@@ -17,6 +17,13 @@ def __init__(self, n):
         self.register_buffer("running_var", torch.ones(n))
 
     def forward(self, x):
+        # Cast all fixed parameters to half() if necessary
+        if x.type() == torch.half:
```

Comment:
If we need to keep this, I think the preferred solution is to do `x.dtype == torch.float16`. (`x.type()` is deprecated or kind of )